gateway-release-common/.gitignore

```diff
@@ -0,0 +1,6 @@
+target/
```

Comment:
The top-level `.gitignore` has this:
```
...
target
```
If it was `**/target` then it would be true, but it's not...I also do not think it's causing any harm here, right?

Reply:
We have 4 .gitignore currently . We only really have one top level .gitignore. 2 others are used as directory placeholders. The last one is for the gateway-admin-ui generated stuff.

I don't think another .gitignore in gateway-release-common is necessary. We have target in other module directories that is generated as part of `mvn` and automatically ignored by the top level .gitignore. Not sure why it would be different in this case.

Reply:
Will check this out...

Reply:
You are right; I'll remove the `.gitignore` here.

Reply:
@risdenk Fixed